### DISCUSSION TITLE
Using wild card IP address for local_addrs in swanctl.conf

### DISCUSSION THREAD

**Discussion by gr4yg00s3** [strongswan/strongswan#1090] [Q&A]
Hello, I wanted to know if it is okay to use the wild card IP/IPv6 Address for the `local_addrs` in the IKE connection parameter in swanctl.conf? I ask this from a standards perspective.

**Comment by tobiasbrunner** [comment]
Not sure what you mean exactly, but if you don't configure anything, `local_addrs` defaults to `%any` anyway.

**Reply by gr4yg00s3** [reply]
Yup, I meant is it good practice to leave it as `%any` or to use a wildcard IP?

**Reply by tobiasbrunner** [reply]
Why not?

**Reply by gr4yg00s3** [reply]
I presumed one would want to be as precise as possible wrt the IKE and ESP parameters. But I guess it should be fine
